src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -439,7 +421,7 @@ def decode_latents(self, latents):
         image = self.vae.decode(latents).sample
         image = (image / 2 + 0.5).clamp(0, 1)
         # we always cast to float32 as this does not cause significant overhead and is compatible with bfloat16
-        image = image.cpu().permute(0, 2, 3, 1).float().numpy()
+        # image = image.cpu().permute(0, 2, 3, 1).float().numpy()
```

Comment:
Is the return type different now?

Reply:
yeah now it returns torch tensor here so if the `output_type` is `pt` it can stays in the device